### DISCUSSION TITLE
Q1:How to encrypt SM2 without ASN.1.

### DISCUSSION THREAD

**Discussion by DanielD00** [openssl/openssl#21487] [Q&A]
Please do NOT use issues to ask questions about OpenSSL.

Instead, please use the [Q&A category in Discussions](<https://github.com/openssl/openssl/discussions/new?category=q-a>)
to ask your question.

**Comment by paulidale** [comment]
Padding?

**Comment by tom-cosgrove-arm** [comment]
Was there any elaboration on the question in the main text? It seems to have been overwritten by the "do not use issues to ask questions" boilerplate that was put into the issue

**Comment by tom-cosgrove-arm** [comment]
Ah - I see it is now in https://github.com/openssl/openssl/discussions/21489 - closing this discussion in favour of #21489
